test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -4180,22 +4182,6 @@ BOOST_AUTO_TEST_CASE(evm_exceptions_in_constructor_call_fail)
 	BOOST_CHECK(callContractFunction("test()") == encodeArgs(2));
 }
 
-BOOST_AUTO_TEST_CASE(evm_exceptions_in_constructor_out_of_baund)
```

Comment:
We could perhaps change this to a check that the compiler cannot detect at compile-time, e.g. you can assign 5 to a temporary variable.